README.md

```diff
@@ -5,20 +5,21 @@ I am releasing the source code (.tex) under GPL license. I
 retain ownership of all drawings and only provide them so the
 overall thing is compilable.
 
-To compile:
+To compile on Mac OS X:
  - Install MacTeX library (for epstopdf and pdflatex):
     - via installer: https://tug.org/mactex/mactex-download.html
     - or via Homebrew, with GUI:    `brew cask install mactex`
     - or via Homebrew, without GUI: `brew cask install mactex-no-gui`
- - Install Inkscape: https://inkscape.org/download/
- - Run: `./make.sh`
+ - Install Inkscape:
+    - https://inkscape.org/download/
+ - Run:
+    - `./make.sh`
 
-__Note:__ this has only been tested on Mac OS X. You will need to adjust the
-build script a little bit and probably fix case-sensitive path on Linux
-but I believe it should work with minimal work.
-
-Debian GNU/Linux dependencies:
- - `sudo apt install inkscape texlive texlive-font-utils texlive-latex-extra`
+To compile on Debian GNU/Linux and derivatives (Ubuntu):
+ - Install Inkscape and TeX Live library (for epstopdf and pdflatex):
+    - `sudo apt install inkscape texlive texlive-font-utils texlive-latex-extra`
```

Comment:
Can we also add the command to install inkscape while we are at it?

Reply:
Inkscape is already in it for Debian/Ubuntu...
But not for Mac OS X because `brew cask install inkscape` fails on xquartz dependency (which is in fact not needed for CLI)

Reply:
Do you mean the command to install Inkscape on Mac?
- For Linux, Inkscape is already installed by the command currently in your README
- For Mac, Inkscape may be installed by `brew cask install xquartz && brew cask install inkscape` but I didn't install it that way: it requires XQuartz installation when Inkscape installed by .pkg doesn't need it at install time. And XQuartz is in fact not needed for CLI nor to build gebbwolf3.